src/keys.rs

```diff
@@ -227,21 +227,21 @@ type IssuanceAuth = SpendAuth;
 ///
 /// [orchardkeycomponents]: https://zips.z.cash/protocol/nu5.pdf#orchardkeycomponents
 #[derive(Debug, Copy, Clone)]
-pub struct IssuanceKey([u8; 32]);
+pub struct IssuanceMasterKey([u8; 32]);
```

Comment:
the documentation should point to our version of the spec with the write paragraph link.

Reply:
Please make sure that all key components are properly documented.